contracts/implementations/ChildGauge.vy

```diff
@@ -99,15 +106,13 @@ integrate_inv_supply: public(HashMap[uint256, uint256])
 integrate_inv_supply_of: public(HashMap[address, uint256])
 
 # For tracking external rewards
-reward_count: public(uint256)
-reward_tokens: public(address[MAX_REWARDS])
-reward_data: public(HashMap[address, Reward])
+reward_data: public(DynArray[RewardData, MAX_REWARDS])
```

Comment:
Use id.

You should've do some work with previous MAX_REWARDS too. It's still counted to be used one reward per token, and possible to add more if something messed up.

I did add claimable/claimed per token but not possible to make same RewardData per token because they can have different periods. Can do DynArray of these campaigns